app/serializers/event.js

```diff
@@ -13,10 +13,17 @@ export default ApplicationSerializer.extend(CustomPrimaryKeyMixin, {
 
   serialize() {
     const json = this._super(...arguments);
-    delete json.data.relationships['general-statistics'];
-    delete json.data.relationships['order-statistics'];
+
+    const { relationships } = json.data;
```

Comment:
```suggestion
    const { relationships } = json.data;
    // We are deleting read only relationships so that they don't
    // break the server. Since these relationships are not always present,
    // we catch and ignore the error
```